pkg_defs/Chandra.Maneuver/build.sh

```diff
@@ -1 +1 @@
-pip install --no-deps --verbose --no-binary :all: --no-index --egg .
+python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
I don't care if these changes are in build.sh or if the script is moved to meta.yaml but wasn't sure which way we wanted to go for this.

Reply:
Also here I do not understand exactly what you mean. I made the change to be minimally invasive, but if you prefer another way, we can do that.

Reply:
Eventually all the `build.sh` are going to be deleted as in shiny and use `script:` in `meta.yaml`.  This is the easier way to do for also supporting Windows builds.  But the changes here are fine now for getting this release out.